### PR TITLE
doc/cephadm: Restoring the MON quorum

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Fixes: https://tracker.ceph.com/issues/49214
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/cephadm/troubleshooting.rst [hunk 1]
```diff
@@ -176,3 +175,28 @@ To access the admin socket, first enter the daemon container on the host::
...
+
+In case the Ceph MONs cannot form a quorum, cephadm is not able
+to manage the cluster, until the quorum is restored. 
+
+In order to restore the MON quorum, remove unhealthy MONs
+form the monmap by following these steps:
+
+1. Stop all MONs. For each MON host::
+
+   ssh {mon-host}
```
Comment (jmolmo) [inline doc/cephadm/troubleshooting.rst:191]: In the preview both commands appear in the same line

File: doc/cephadm/troubleshooting.rst [hunk 2]
```diff
@@ -176,3 +175,28 @@ To access the admin socket, first enter the daemon container on the host::
...
+   cephadm unit --name mon.`hostname` stop
+
+
+2. Identify a surviving monitor and log in to that host::
+
+   ssh {mon-host}
```
Comment (jmolmo) [inline doc/cephadm/troubleshooting.rst:197]: In the preview both commands appear in the same line

File: doc/cephadm/troubleshooting.rst [hunk 3]
```diff
@@ -176,3 +175,28 @@ To access the admin socket, first enter the daemon container on the host::
 
     [root@mon1 ~]# cephadm enter --name <daemon-name>
     [ceph: root@mon1 /]# ceph --admin-daemon /var/run/ceph/ceph-<daemon-name>.asok config show
+
+
+Restoring the MON quorum
+------------------------
+
+In case the Ceph MONs cannot form a quorum, cephadm is not able
```
Comment (jmolmo) [inline doc/cephadm/troubleshooting.rst:183]: A "ceph orch pause"  before and after execute the procedure could be healthy.

**Review by jmolmo — CHANGES_REQUESTED** [review]
What about to pause orchestrator while the monitors are "dancing" ?  Format the commands in different lines

File: doc/cephadm/troubleshooting.rst (hunk 3 above)
Comment (sebastian-philipp) [reply doc/cephadm/troubleshooting.rst:183]: if the MONs are not in quorum, there is nothing you can do about it. `ceph orch pause` will fail. `ceph config-key set mgr/cephadm/pause` will also fail.

File: doc/cephadm/troubleshooting.rst (hunk 2 above)
Comment (sebastian-philipp) [reply doc/cephadm/troubleshooting.rst]: fixed

File: doc/cephadm/troubleshooting.rst (hunk 1 above)
Comment (sebastian-philipp) [reply doc/cephadm/troubleshooting.rst]: fixed

**Review by jmolmo — APPROVED** [review]

**Pull request merged**